pkg/acsengine/params_k8s.go

```diff
@@ -96,17 +96,6 @@ func assignKubernetesParameters(properties *api.Properties, parametersMap params
 				CloudProviderRateLimitBucket: kubernetesConfig.CloudProviderRateLimitBucket,
 			})
 			addValue(parametersMap, "kubeClusterCidr", kubernetesConfig.ClusterSubnet)
-			if !properties.IsHostedMasterProfile() {
```

Comment:
Same comment as above.

Reply:
also same as above w/ respect to addons refactor